src/lib/getNavigationEntry.ts

```diff
@@ -18,7 +18,7 @@ export const getNavigationEntry = ():
   | PerformanceNavigationTiming
   | undefined => {
   return (
-    window.performance &&
+    self.performance &&
```

Comment:
oh, you're actually changing it in code as well, in which case `self` makes more sense for old browser support. Buy why now? I'm so curious.

Reply:
I standardized on `self` because both were used throughout the project. Though looking at it closer now, I see `self` was only ever used in test files, so maybe it doesn't matter?

Reply:
Ah, yeah, makes sense. Probably doesn't matter, but also fine either way. I had no idea `self` has been around since at least IE6.  I thought it was a web-worker-era addition.